geofmureithi/wasm-javascript-exercises
# Adding a scoreboard to the wasm exercises

# Scoreboard Feature ?
Hey was wondering if you'd be interested in this feature so that next time you do a similar presentation, there can be a website on a projector/screen showing how the teams are ranking depending on which exercises they've finished. When a team pushes a solution that passes an exercise, the points are stored in a database via GH Actions, and the website displays running totals on a scoreboard.

## URL
Here's the simple scoreboard. in keeping with the Rust theme, it's served by Actix Web and uses Yew for server side rendering (we could change to Hirola :)): https://wasm-javascript-exercises-production.up.railway.app/scoreboard

## How to test it
If you want to test it, I think you need clone the repo from my fork coz of secrets stored in my repo but I'm not sure. Then:
- Push a passing test
- Refresh the scoreboard link above to see the updated scoreboard

If you like the idea, maybe we could make the scoreboard it's own repo so it's separate from the repo that contains the exercises?
